Index: test_1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_1.py	(date 1573811798000)
+++ test_1.py	(date 1573815943212)
@@ -42,7 +42,7 @@
     for info in ip_same:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                                     ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
@@ -52,7 +52,7 @@
     for info in ip_diff:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                 ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
Index: test_2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_2.py	(date 1573811798000)
+++ test_2.py	(date 1573815974159)
@@ -42,7 +42,7 @@
     for info in ip_same:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                                     ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
@@ -52,7 +52,7 @@
     for info in ip_diff:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                 ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
Index: test_3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_3.py	(date 1573811798000)
+++ test_3.py	(date 1573815974138)
@@ -42,7 +42,7 @@
     for info in ip_same:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                                     ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
@@ -52,7 +52,7 @@
     for info in ip_diff:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                 ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
Index: test_4.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_4.py	(date 1573811798000)
+++ test_4.py	(date 1573815974064)
@@ -42,7 +42,7 @@
     for info in ip_same:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                                     ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
@@ -52,7 +52,7 @@
     for info in ip_diff:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                 ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- main.py	(date 1573811798000)
+++ main.py	(date 1573815931826)
@@ -26,7 +26,7 @@
     for info in ip_same:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                                     ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
@@ -36,7 +36,7 @@
     for info in ip_diff:
         domain, ip_final = info[0], info[1]
         if domain:
-            sql = """INSERT INTO ip_mapping_relations (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
+            sql = """INSERT INTO ip_mapping_relations_real (ip_origin, domain, ip) VALUES ('{ip_origin}', '{domain}', '{ip}') """.format(
                 ip_origin=ip, domain=domain, ip=';'.join(ip_final)
             )
             print sql
Index: scritps.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- scritps.py	(date 1573958556563)
+++ scritps.py	(date 1573958556563)
@@ -0,0 +1,16 @@
+"""
+Scripts
+"""
+
+from Database.database import DB
+
+
+def get_info():
+    DB().db.cursor().execute()
+    info = DB().db.cursor().fetchall()
+    for i in info:
+        print i
+
+
+if __name__ == '__main__':
+    get_info()
Index: Database/database.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Database/database.py	(date 1573811798000)
+++ Database/database.py	(date 1573959198222)
@@ -17,6 +17,12 @@
         try:
             self.db = MySQLdb.connect(**mysql_conf)
             self.cursor = self.db.cursor()
+            self.cursor.execute("select ip_origin, domain, ip from ip_mapping_relations_real where ip_origin != ip;")
+            for i in self.cursor.fetchall():
+                if i[2] == '':
+                    print i[0], i[1], 'Null'
+                else:
+                    print i[0], i[1], i[2]
         except Exception as e:
             print e
 
@@ -29,5 +35,5 @@
 
 
 if __name__ == '__main__':
-    DB().insert("sss")
+    DB()
     pass
